Postman Test Suite/Ed-Fi ODS-API Integration Test Suite ResponseTests.postman_collection.json

```diff
@@ -5646,33 +5699,21 @@
-													"    ],",
-													"    \"$.classPeriods[1].classPeriodReference.schoolId\": [",
-													"      \"All values supplied for 'schoolId' must match. Review all references (including those higher up in the resource's data) and align the following conflicting values: '777', '999'\"",
-													"    ],",
-													"    \"$.classPeriods[2].classPeriodReference.schoolId\": [",
-													"      \"All values supplied for 'schoolId' must match. Review all references (including those higher up in the resource's data) and align the following conflicting values: '777', '999'\"",
-													"    ],",
-													"    \"$.classPeriods[3].classPeriodReference.schoolId\": [",
-													"      \"All values supplied for 'schoolId' must match. Review all references (including those higher up in the resource's data) and align the following conflicting values: '777', '999'\"",
-													"    ],",
-													"    \"$.classPeriods[4].classPeriodReference.schoolId\": [",
-													"      \"All values supplied for 'schoolId' must match. Review all references (including those higher up in the resource's data) and align the following conflicting values: '777', '999'\"",
-													"    ],",
-													"    \"$.classPeriods[5].classPeriodReference.schoolId\": [",
-													"      \"All values supplied for 'schoolId' must match. Review all references (including those higher up in the resource's data) and align the following conflicting values: '777', '999'\"",
-													"    ],",
-													"    \"$.classPeriods[6].classPeriodReference.schoolId\": [",
-													"      \"All values supplied for 'schoolId' must match. Review all references (including those higher up in the resource's data) and align the following conflicting values: '777', '999'\"",
-													"    ]",
-													"  });",
+													"  // We should see at least 6 validation messages (for each member of the child collection)",
+													"  // but may also produce one for: \"$.schoolReference.schoolId\"",
+													"  pm.expect(Object.keys(problemDetails.validationErrors).length).to.be.gte(7);",
+													"  pm.expect(Object.keys(problemDetails.validationErrors).length).to.be.lte(8);",
+													"",
+													"  // Verify entries for each of the children allowing for 2 or 3 values (of 777, 888 and/or 999) as valid entries.",
+													"  for (var i = 0; i <= 6; i++) {",
+													"    pm.expect(problemDetails.validationErrors[`$.classPeriods[${i}].classPeriodReference.schoolId`])",
+													"      .to.match(/All values supplied for 'schoolId' must match. Review all references \\(including those higher up in the resource's data\\) and align the following conflicting values: (('777'|'888'|'999')(,\\s)?){2,3}/i);",
```

Comment:
`777`, `888` and `999` are all values that could appear in these messages even though the current Ed-Fi ODS API doesn't include all values in all messages -- it would not be considered incorrect if they were included.